pkg/agent/openflow/pipeline.go

```diff
@@ -1372,6 +1395,39 @@ func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.Tab
 	return flows
 }
 
+// conjunctionActionRejectFlow generates the flow to mark the packet to be rejected if policyRuleConjunction ID is matched.
+// Any matched flow will be dropped in corresponding metric tables.
+func (c *client) conjunctionActionRejectFlow(conjunctionID uint32, tableID binding.TableIDType, priority *uint16, enableLogging bool) binding.Flow {
```

Comment:
There is only few difference between conjunctionActionRejectFlow and conjunctionActionDropFlow, can we use some parameter and have only one method?

Reply:
Changed.